pkg/sentry/fsimpl/fuse/dev.go

```diff
@@ -339,22 +339,20 @@ func (fd *DeviceFD) Seek(ctx context.Context, offset int64, whence int32) (int64
 
 // sendResponse sends a response to the waiting task (if any).
 func (fd *DeviceFD) sendResponse(ctx context.Context, fut *futureResponse) error {
-	// See if the running task need to perform some action before returning.
-	// Since we just finished writing the future, we can be sure that
-	// getResponse generates a populated response.
-	if err := fd.noReceiverAction(ctx, fut.getResponse()); err != nil {
-		return err
-	}
+	// Signal the task waiting on a response if any.
+	defer close(fut.ch)
```

Comment:
Will a malicious FUSE daemon manage to send multiple responses to the same request and thus crush the sentry? I see the write() acquires the lock but is there any chance this can happen?